### ISSUE TITLE
Add figwheel nrepl middleware

### ISSUE THREAD

**Issue by lvh** [RackSec/desdemona#72]
Appropriate middleware might be:

```
["cider.nrepl/cider-middleware"
 "refactor-nrepl.middleware/wrap-refactor"
 "cemerick.piggieback/wrap-cljs-repl"]
```

Although it's not clear what happens when some of that's missing (e.g `wrap-refactor`). If we do this before #71 that will make #71 a bit easier since we won't have to document how to do `(use 'figwheel-sidecar.repl-api) (figwheel-sidecar.repl-api/cljs-repl)`.

**Comment by lvh** [comment]
(We should also check if figwheel has actually started supporting this properly.)

**Comment by lvh** [comment]
This was closed in #72.
